### ISSUE TITLE
Cascader bind with value display issue

### ISSUE THREAD

**Issue by JitenPatoliya** [ant-design-blazor/ant-design-blazor#702]
### Describe the bug
When I bind cascader with value it shows like  "Size 1/ Size 1", it should only show "Size 1"
I made childern property null when defining cascadernode, but no luck

### Steps to reproduce (please include code)

<div>
    <Cascader Options="@sizeNodes" Value="@currValue" Size="small" Style="margin:15px 0"></Cascader>
    <Cascader Options="@sizeNodes" @bind-Value="@currValue" Size="small" Style="margin:15px 0"></Cascader>
</div>

@code {
    List<CascaderNode> sizeNodes = GetSizeNodes();
    string currValue = "1";
    private static List<CascaderNode> GetSizeNodes()
    {
        List<CascaderNode> list = new List<CascaderNode>() {
        new CascaderNode()
        {
            Value = "1",
            Label = "Size1",
        }, new CascaderNode()
        {
            Value = "2",
            Label = "Size2",
        }, new CascaderNode()
        {
            Value = "3",
            Label = "Size3",
        }};
        return list;
    }
}

**Comment by ElDiddi** [comment]
Hello JitenPatoliya,

the following code works as expected.

```
<div>
    <Cascader Options="@(_basicNodes2)" DefaultValue="2" @bind-Value="_value"></Cascader>
</div>

@code{
    List<CascaderNode> _basicNodes2 = GetBasicNodes2();
    string _value;

    private static List<CascaderNode> GetBasicNodes2()
    {
        List<CascaderNode> list = new List<CascaderNode>() {
            new CascaderNode()
            {
                Value = "1",
                Label = "Size1",
            }, new CascaderNode()
            {
                Value = "2",
                Label = "Size2",
            }, new CascaderNode()
            {
                Value = "3",
                Label = "Size3",
            }};

        return list;
    }
}
```

**Comment by JitenPatoliya** [comment]
Hello ElDiddi,

missed few lines of code when posting original.
Here's the full code producing problem as described (using example you replied).

```
<div>
    <Cascader Options="@(_basicNodes2)" @bind-Value="_value"></Cascader>
</div>

@code{
    List<CascaderNode> _basicNodes2 = GetBasicNodes2();
    string _value = "2";

    private static List<CascaderNode> GetBasicNodes2()
    {
        List<CascaderNode> list = new List<CascaderNode>() {
            new CascaderNode()
            {
                Value = "1",
                Label = "Size1",
            }, new CascaderNode()
            {
                Value = "2",
                Label = "Size2",
            }, new CascaderNode()
            {
                Value = "3",
                Label = "Size3",
            }};

        return list;
    }
}
```

**Comment by ElderJames** [comment]
We have fixed it, please try lastest 0.7 nightly version.
